Dockerfile

```diff
@@ -310,10 +310,29 @@ RUN chown -R $USER /work/rust-optee-trustzone-sdk/optee-qemuv8-3.7.0/build/build
 WORKDIR /
 RUN ./cleanup.sh
 
+FROM build_sgx as build_icecap
```

Comment:
Do we want to use the sgx build as the base for icecap? Doesn't it have things we don't need for icecap?